### ISSUE TITLE
Código de conducta 

### ISSUE THREAD

**Issue by penyaskito** [SevillaDevelopers/sevilladevelopers.github.io#15]
Creo que, aparte de tener un código de conducta para los eventos que se organicen, es necesario tener un código de conducta para la comunidad en sí  (grupo de fb, otros medios)

**Comment by israbr** [comment]
Te refieres hacerlo aun mas explicito?
En la pagina hay esto:
_"Ser una persona educada (No siendo un troll, respetando otras opiniones y tratar siempre de ayudar. Aquí tienes más información al respecto"_
donde *Aqui* te lleva al código de conducta de berlin

**Comment by penyaskito** [comment]
El PR anterior lo hace más explícito donde comentas. Creo que, aparte, sería necesario tener contactos en caso de que alguien quisiera comunicarse en privado, y que estuviera en castellano. Yo propondría tenerlo como una página del sitio en castellano con esos datos, y que en el grupo de facebook estuviera enlazado en la descripción y en el post fijado.

**Comment by israbr** [comment]
Fixed in #18

**Comment by penyaskito** [comment]
"Creo que, aparte, sería necesario tener contactos en caso de que alguien quisiera comunicarse en privado, y que estuviera en castellano. Yo propondría tenerlo como una página del sitio en castellano con esos datos, y que en el grupo de facebook estuviera enlazado en la descripción y en el post fijado."
IMHO aún no cerraría esto.

**Comment by israbr** [comment]
TOOOODA LA RAZON DEL MUNDO.
Se me paso esa parte.

**Comment by danibishop** [comment]
Si alguno me da un ejemplo de lo que queréis poner, me asigno esto. Es que no me entero muy bien con tu comentario, @penyaskito :(

**Comment by miguelbemartin** [comment]
Creo que quedó cerrada con la #18

**Comment by danibishop** [comment]
Nope. Penyaskito e Isra reabrieron el ticket por algo... que es lo que no
entiendo :D



[image: --]

Dani 'bISHOP' Ramírez-Martínez
[image: http://]about.me/daniramirez
<http://about.me/daniramirez?promo=email_sig>


2018-04-28 20:40 GMT+02:00 Miguel Ángel Martín <notifications@github.com>:

**Comment by penyaskito** [comment]
Supongamos que alguien sufre algún tipo de incidencia y quiere reportarla/comunicarla.
¿Cómo puede contactar? ¿Dónde puedo acudir?
